lib/game/GameApi.js

```diff
@@ -1,104 +1,110 @@
+        }
+        return data;
+    });
+};
+const newGame = async (playerName, quickGame) => {
+    if (quickGame) {
+        const games = await pendingGames();
+        if (games.length > 0) {
+            return joinGame(playerName, games[0].id);
+        }
+    }
+    const playerId = uuidv4();
+    const requestOptions = {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ playerName, playerId, quickGame: quickGame }),
+    };
+
+    return fetch(`${CONSTANTS.BASE_URL}/game`, requestOptions).then(
+        async (res) => {
             const data = await res.json();
             if (!res.ok) {
                 const error = (data && data.message) || res.status;
-                return Promise.reject(new Error(error));
-            }
-            return data;
-        });
-    },
-    joinGame: (playerName, gameId) => {
-        const requestOptions = {
-            method: "POST",
-            headers: {
-                "Content-Type": "application/json",
-            },
-            body: JSON.stringify({ playerName }),
-        };
-        return fetch(
-            `${CONSTANTS.BASE_URL}/game/${gameId}/join`,
-            requestOptions
-        ).then(async (res) => {
-            if (!res.ok) {
-                return Promise.reject("une erreur!");
+                return Promise.reject(error);
             }
-            return await res.json();
-        });
-    },
-    getGame: (gameId) => {
-        return fetch(`${CONSTANTS.BASE_URL}/game/${gameId}`).then(
-            async (res) => {
-                const data = await res.json();
-                if (!res.ok) {
-                    const error = (data && data.message) || res.status;
-                    return Promise.reject(error);
-                }
-                return data;
-            }
-        );
-    },
+            return [data, playerId];
+        }
+    );
+};
+const joinGame = (playerName, gameId) => {
+    const playerId = uuidv4();
+    const requestOptions = {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ playerName, playerId }),
+    };
+    return fetch(
+        `${CONSTANTS.BASE_URL}/game/${gameId}/join`,
+        requestOptions
+    ).then(async (res) => {
+        if (!res.ok) {
+            return Promise.reject("une erreur!");
+        }
+        return await [await res.json(), playerId];
+    });
+};
+const getGame = (gameId) => {
+    return fetch(`${CONSTANTS.BASE_URL}/game/${gameId}`).then(async (res) => {
+        const data = await res.json();
+        if (!res.ok) {
+            const error = (data && data.message) || res.status;
+            return Promise.reject(error);
+        }
+        return data;
+    });
 };
 
-export default gameApi;
+export default { getGame, pendingGames, newGame, joinGame, tilt, replace };
```

Comment:
Why don't you export each function directly ?

Reply:
I'm not very used to export multiple functions, so I ended up exporting like this. What are the best practices regarding multiple function export?

Reply:
```js
export const myFunction = () => { ... };

// later
import { myFunction } from './myFile';
```